### PR TITLE
Fix #192 six-sidebar text overflow

### LINKED ISSUE #192: Sidebar can overflow when text is too long
### Six Webcomponents version

4.0.4

### Steps to reproduce

Add long text to a sidebar item group

### What is expected?

Either the sidebar should extend automatically or the text should wrap

### What is actually happening?

image

### System Info

_No response_

### Any additional comments?

_No response_

### PR DESCRIPTION
**Pull request by belahorvath**
### 🔗 Linked issue

#192 
### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [x] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

The text will now overflow in the sidebar when it reaches the border. I did not want to extend the sidebar, this would lead to some quite large sidebars that look weird in my opinion.

Chat me up if you want to talk about that :)

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [ ] It's submitted to the `main` branch
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] I have updated the documentation accordingly.
- [x] All tests are passing
- [ ] New/updated tests are included
- [x] I have updated the "upcoming" section inside _docs/changelog.md_ explaining the changes I contributed

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

**Review by davidemarcoli — DISMISSED** [review]
Looks good!

File: libraries/ui-library/src/components/six-sidebar-item-group/six-sidebar-item-group.scss [hunk 1]
```diff
@@ -23,6 +23,8 @@
 
 .six-sidebar-details__header {
   display: flex;
+  overflow-wrap: anywhere;
+  align-items: center;
```
Comment (ibirrer) [inline libraries/ui-library/src/components/six-sidebar-item-group/six-sidebar-item-group.scss:27]: Thanks for the fix :-). Looks good, but I would keep the default alignment. I look a bit strange if the icons are not aligned with the first line:

Center
image

Default
image

**Review by ibirrer — CHANGES_REQUESTED** [review]

File: libraries/ui-library/src/components/six-sidebar-item-group/six-sidebar-item-group.scss (hunk 1 above)
Comment (belahorvath) [reply libraries/ui-library/src/components/six-sidebar-item-group/six-sidebar-item-group.scss:27]: I wasn't too sure either, I agree that it looks clearer if the icon stays on the first line. Will adapt this tomorrow :)

Comment (belahorvath) [reply libraries/ui-library/src/components/six-sidebar-item-group/six-sidebar-item-group.scss]: Removed the center alignment

**Review by ibirrer — APPROVED** [review]

**Pull request merged**